src/hazelcore/Statics_Exp.re

```diff
@@ -285,7 +288,8 @@ and ana_operand =
   | Inj(InHole(WrongLength, _), _, _)
   | Case(StandardErrStatus(InHole(WrongLength, _)), _, _) =>
     ty |> HTyp.get_prod_elements |> List.length > 1 ? Some() : None
```

Comment:
not sure whats going on here -- case should never have a WrongLength hole on it.

Reply:
I believe this is a minor bug in `dev` unrelated to this PR -- this should always return None because these forms should never have WrongLength holes on them. we really should special case WrongLength errors to be an argument to the Comma operator in a BinOp...